common/msg/tensors.msg

```diff
@@ -0,0 +1 @@
+std_msgs/UInt8MultiArray[] tensors
```

Comment:
Is this still safe and valid for tensors with float/double/int32 and so on?

Reply:
std_msgs/UInt8MultiArray includes a container of vector<uint8_t>  (i.e., byte array) and a header describing what this vector actually is. In order to convert such byte vector to other type of data, the name of the actual type of published tensor is embedded in the header using tensor_element_typename[]'.

In short, IMO, this is safe way to handle all of the types of tensor but endian (ROS message is serialized with little-endian order) or data alignment issues could be addressed.